src/App.vue

```diff
@@ -241,10 +239,42 @@
 </style>
 
 <style lang="scss" scoped>
-	// modal layout
+
+	.modal-mask {
+		background-color: rgba(0, 0, 0, 0.7);
+
+		& /deep/ .modal-wrapper .modal-container {
+			display: flex;
+			flex-direction: column;
+			height: 95% !important;
+			width: 95% !important;
+			max-width: 900px;
+			max-height: 650px !important;
+			position: relative;
+		}
+
+		& /deep/ .modal-body {
```

Comment:
You're editing the default styling, but maybe the component needs a fix?
Did you find anything that was maybe weird or unfit in the component itself? :)

Reply:
Fixed.